common/buildings/wc_special_buildings.txt

```diff
@@ -1791,6 +1791,166 @@ falanaar_tunnels_01 = {
 	}
 }
 
+neltharions_lair_01 = {
+	#placeholder?
+	type_icon = "icon_structure_onyxias_lair.dds"
+
+	province_modifier = {
+		garrison_size = 0.1
+		travel_danger = 10
+		fort_level = 2
+	}
+
+	county_modifier = {
+		levy_size = 0.3
+		tribal_holding_build_speed = -0.5
+	}
+
+	character_modifier = {
+		heavy_infantry_toughness_mult = 0.05
+		army_maintenance_mult = -0.05
+		monthly_prestige_gain_mult = 0.1
+	}
+
+	can_construct_potential = {
+		always = no
+	}
+
+	type = special
+	flag = travel_point_of_interest_martial
+
+	is_enabled = {
+		scope:holder = {
+			OR = {
+				has_trait = creature_black_dragon
+				has_trait = creature_drogbar
+				faith = faith:neltharism
+				faith = faith:drogbar_religion
+				has_culture = culture:drogbar 
+				has_culture = culture:black_dragon
+			}
+		}
+	}
+
+	next_building = neltharions_lair_02
+}
+
+neltharions_lair_02 = {
+	type_icon = "icon_structure_onyxias_lair.dds"
+	
+	type = special
+	flag = travel_point_of_interest_wonder
+	construction_time = slow_construction_time
+
+	cost_gold = 750
+	
+	character_modifier = {
+		tribal_holding_build_speed = -1
+		monthly_prestige_gain_mult = 0.1
+		army_maintenance_mult = -0.01
+		heavy_infantry_toughness_mult = 0.15
+		defender_advantage = 3
+	}
+
+	province_modifier = {
+		fort_level = 2
+		travel_danger = 5
+	}
+
+	county_modifier = {
+		levy_size = 0.4
+	}
+
+	is_enabled = {
+		scope:holder = {
+			OR = {
+				has_trait = creature_black_dragon
+				has_trait = creature_drogbar
+				faith = faith:neltharism
+				faith = faith:drogbar_religion
+				has_culture = culture:drogbar 
+				has_culture = culture:black_dragon
+			}
+		}
+	}
+}
+
+vault_of_the_wardens_01 = {
```

Comment:
I've put Vault of the Wardens into the modifier cost calculator and it is spitting out the total cost at 173. I've had to supplement men_at_arms_maintenance with army_maintenance as the modifier doesn't consider it yet, however most other special buildings on average sit around a cost of 40. Needs to be nerfed a lot

Reply:
army_maintenance_mult brings it up a lot, just removed  men_at_arms_maintenance all together and replaced it with army_maint. now it sits at 37.9 on sheet